src/core/component.test.ts

```diff
@@ -1371,6 +1371,34 @@ describe("template", () => {
         expect(vm.$children[0].$children.length).toEqual(0);
     });
 
+    it("destroyed component should remove updated subscription", (done) => {
```

Comment:
```suggestion
    it("destroyed component should remove subscriptions", (done) => {
```

Are there any tests checking the same thing? I mean subscriptions removal